pkg/shim/scheduler.go

```diff
@@ -282,6 +268,14 @@ func (ss *KubernetesShim) run() {
 	ss.apiFactory.Start()
 }
 
+func (ss *KubernetesShim) enterState(event *fsm.Event) {
+	log.Logger.Info("scheduler shim state",
+		zap.String("currentState", event.Src),
+		zap.String("targetState", event.Dst),
+		zap.String("event", event.Event),
+		zap.String("currentSchedulerState", ss.GetSchedulerState()))
+}
```

Comment:
This is a double up: we have current and target state already (currentAppState).
Log at a debug level and with same info order: shim identification, source, destination and event.